src/Stryker.CLI/Stryker.CLI/CliInputParser.cs

```diff
@@ -0,0 +1,244 @@
+                    _ => true
+                };
+            }
+        }
+
+        private static bool ParseNoValue(IInputDefinition<bool?> strykerInput)
+        {
+            strykerInput.SuppliedInput = true;
+            return true;
+        }
+
+        private static bool ParseSingleStringValue(CommandOption cliInput, IInputDefinition<string> strykerInput)
+        {
+            strykerInput.SuppliedInput = cliInput.Value();
+            return true;
+        }
+
+        private static bool ParseSingleIntValue(CommandOption cliInput, IInputDefinition<int?> strykerInput)
+        {
+            if (int.TryParse(cliInput.Value(), out int value))
+            {
+                strykerInput.SuppliedInput = value;
+            } else
+            {
+                throw new StrykerInputException($"Unexpected value for argument {cliInput.LongName}:{cliInput.Value()}. Expected type to be integer");
+            }
+            return true;
+        }
+
+        private static bool ParseSingleOrNoValue(IInputDefinition strykerInput, CommandOption cliInput, StrykerInputs strykerInputs)
+        {
+            switch (strykerInput)
+            {
+                // handle single or no value inputs
+                case SinceInput sinceInput:
+                    sinceInput.SuppliedInput = true;
+                    _strykerInputs.SinceTargetInput.SuppliedInput = cliInput.Value();
+                    break;
+                case WithBaselineInput withBaselineInput:
+                    withBaselineInput.SuppliedInput = true;
+                    _strykerInputs.BaselineProviderInput.SuppliedInput = cliInput.Value();
+                    break;
+            }
+            return true;
+        }
+
+        private static bool ParseMultiValue(CommandOption cliInput, IInputDefinition<IEnumerable<string>> strykerInput)
+        {
+            strykerInput.SuppliedInput = cliInput.Values;
+
+            return true;
+        }
+
+        private static IInputDefinition GetStrykerInput(CommandOption cliInput) => _cliInputs[cliInput.LongName].Input;
+
+        private static void PrepareCliOptions()
+        {
+            AddCliInput(_strykerInputs.DevModeInput, "dev-mode", null, optionType: CommandOptionType.NoValue);
+            AddCliInput(_strykerInputs.ConcurrencyInput, "concurrency", "c", argumentHint: "number");
+            AddCliInput(_strykerInputs.SolutionPathInput, "solution", "s", argumentHint: "file-path");
+
+            AddCliInput(_strykerInputs.ReportersInput, "reporter", "r", optionType: CommandOptionType.MultipleValue);
+            AddCliInput(_strykerInputs.MutateInput, "mutate", "m", optionType: CommandOptionType.MultipleValue, argumentHint: "glob-pattern");
+            AddCliInput(_strykerInputs.ThresholdBreakInput, "threshold-break", "b", argumentHint: "0-100");
```

Comment:
```suggestion
            AddCliInput(_strykerInputs.ThresholdBreakInput, "break-at", "b", argumentHint: "0-100");
```

How about, to keep it more in line with the other change to options we've been making (since, with-baseline) to call it break-at in the command line? @richardwerkman

Reply:
Actually also a good option